app/assets/stylesheets/color.scss

```diff
@@ -0,0 +1,7 @@
+/* forms */
+
+input, textarea, select, .uneditable-input {
+  border: 1px solid #bbb;
```

Comment:
#bbb thay bằng biến màu, chứ có phải anh bảo copy toàn bộ csss của nó vào file này đâu